docs/entrypoints.md

```diff
@@ -0,0 +1,56 @@
+# Entrypoints
+
+This document describes the various ways that weave as a binary gets launched, and where and how it processes options.
+
+The goal is to enable maintainers to modify functionality without having to reverse engineer the code to find all of the places you can start a weave binary and what it depends upon.
+
+## Components
+### weaveutil
```

Comment:
I consider `weaver` to be more primary :slightly_smiling_face: 

So I would insert:

### weaver
`weaver` is the primary binary: it sets up the `weave` bridge to which containers are attached; manages IP address allocation, serves DNS requests and does many more things.

Since version 2.0, `weaver` also bundles the Docker Plugin (legacy version) and Docker API Proxy which used to be shipped as separate containers.